changelog/2520.feature.rst

```diff
@@ -0,0 +1 @@
+Added the authority property to the Url class
```

Comment:
Can we add a note here about how this property should be used instead of `netloc`.